sys/vane/jael.hoon

```diff
@@ -687,6 +687,11 @@
       ::  save our parent signature (only for moons)
       ::
       =.  sig.own.sub  sig.seed.tac
+      ::  if we're given a snapshot, restore it
```

Comment:
Since we hit this as an issue, maybe worth adding another line of comment saying why it's important that this happens before "our initial public key".

Reply:
Sure